pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Did you run this with a newer version of `pnpm` than v8.5.1? This should probably not be in this PR, but hey, if it's needed – let's get it in there! :)